### ISSUE TITLE
[Accessibility]: Notebooks read-only ARC audit tabindex 

### ISSUE THREAD

**Issue by marisak** [sourcegraph/sourcegraph#35678]
### Audit type

ARC Toolkit

### User journey audit issue

https://github.com/sourcegraph/sourcegraph/issues/35142

### Problem description

From ARC extension: 
> tabindex is used on a non-interactive component in a way that puts it in the tab order. Only active element should be in the tab order

This appears to relate to the body of the notebook, with the stack of notebook blocks. I wonder if this is related to some of the keyboard navigation issues? 

### Expected behavior

Expectation from ARC extension: 
> Remove the tabindex attribute or set it to '-1', OR define the role of the control.

### Assigning labels

- [X] Please give this issue an estimate by applying a label like `estimate/Xd`, where X is the estimated number of days it will take to complete.
- [X] If this issue is specific to a specific Sourcegraph product, please assign the appropriate team label to this issue.
- [ ] If this issue will require input from designers in order to complete, please assign the label `needs-design`.
- [ ] If you are confident that this issue should be fixed by GitStart, please assign the label `gitstart`.

### Owner

I'm unsure who should work on this issue or if it needs more clarification. I will let the Frontend Platform team triage this.

**Comment by sourcegraph-bot-2** [comment]
Heads up @benvenker @lguychard - the "team/search-product" label was applied to this issue.
